### ISSUE TITLE
setup: location permission denied

### ISSUE THREAD

**Issue by ppanero** [inveniosoftware/invenio-cli#88]
When setting up a local / container instance the `location` gets assigned different permissions and therefore the `setup` command needs to be rerun in order to switch between local / container envs. The switch should be transparent.

**Comment by fenekku** [comment]
Hmm, that puts into sharp focus what should be the same and what should differ when running `invenio-cli run` versus `invenio-cli containerize`...

The 2 don't share the same file system, so sharing the files is a bit tricky:
- either we mount a shared folder for them (not a volume anymore)
- either we docker cp them when running `containerize`

In both cases, this is assuming the DB between the 2 is shared correctly which needs to be done first.

Only then, can the potential permission problem be addressed which might be messy...

Depends on #108 , which I want to push to next sprint. So I would vote to push this to next sprint too. Maybe someone has some insights to make this easier?

**Comment by ppanero** [comment]
@fenekku Sounds good to me, is not priority. I am not sure how much it applies to the new CLI workflow either. In the previous one the DB, ES etc were fully shared but then got a perm denied on files. In the new one, since you can wipe (`--force`) when containerize, I'm not sure it applies. Let's postpone it (Moving to March board).

**Comment by fenekku** [comment]
**Grooming**
Potentially nix with #108
